_docs/geyser/anticheat-compatibility.md

```diff
@@ -29,10 +24,5 @@ Partially Compatible (Does not Check/Ignores Bedrock Players)
 
 Incompatible (False Positives on Bedrock Players, no Compatibility at All)
 
-- [AntiCheatReloaded](https://www.spigotmc.org/resources/23799/){:target="_blank"} - Last Updated on 19th August 2021
-- [Negativity V1](https://www.spigotmc.org/resources/48399/){:target="_blank"} - Last Updated on 21st March 2022
 - [NoCheatPlus](https://ci.codemc.io/job/Updated-NoCheatPlus/job/Updated-NoCheatPlus/){:target="_blank"} ([Compat NCP](https://github.com/Updated-NoCheatPlus/CompatNoCheatPlus/){:target="_blank"} is an addon that enables Partial Compatibility) - Last Updated on 19th August 2021
 - [GodsEye](https://www.spigotmc.org/resources/69595/){:target="_blank"} ([GodsEyeGeyserMC](https://github.com/TheDejavu/GodsEyeGeyserMC/releases){:target="_blank"} is an addon that enables Partial Compatibility) (Paid) - Last Updated on 21st March 2022
-- [Kauri](https://www.spigotmc.org/resources/53721/){:target="_blank"} (Paid) - Last Updated on 21st March 2022
-- [Reflex](https://www.spigotmc.org/resources/21122/){:target="_blank"} (Paid) - Last Updated on 19th August 2021
-- [Wraith](https://www.spigotmc.org/resources/66887/){:target="_blank"} (Paid) - Last Updated on 19th August 2021
```

Comment:
Is there an official statement that this is done or just assuming since there's been no updates?

Reply:
I was assuming since there's been no updates but upon checking the github it seems the dev wants to update it. 
So it's still supported.